source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -409,6 +409,9 @@ message TransformationAccessChain {
   // OpAccessChain instruction should be inserted
   InstructionDescriptor instruction_to_insert_before = 4;
 
+  // Additional fresh ids, required to clamp index variables
+  repeated UInt32Pair fresh_ids_for_clamping = 5;
```

Comment:
I would clarify what each id is used for. In fact, I would add a large comment either here, or just below `message TransformationAccessChain` explaining how clamping instructions may also be added.

Reply:
Done. Check that it's clear.